Client/Frontend/Popup/AlertPopupView.swift

```diff
@@ -100,9 +120,20 @@ class AlertPopupView: PopupView {
         messageLabelFrame.origin.y = rint(titleLabelFrame.maxY + kPadding * 1.5 / 2.0 * resizePercentage)
         messageLabel.frame = messageLabelFrame
         
+        var textFieldFrame = textField?.frame ?? CGRect.zero
+        var maxY = !(messageLabel.text?.isEmpty ?? true) ? messageLabelFrame.maxY : titleLabelFrame.maxY
```

Comment:
replace `!(messageLabel.text?.isEmpty ?? true)` with `messageLabel.text?.isEmpty == false`, it's a bit simpler to read

Reply:
I think even easier would be to reverse this:
```suggestion
        var maxY = messageLabel.text?.isEmpty == true ? titleLabelFrame.maxY : messageLabelFrame.maxY
```